docs/usage/configuration-options.md

```diff
@@ -2215,6 +2224,11 @@ The following would match any file in directories starting with `app/`:
 }
 ```
 
+<!-- prettier-ignore -->
+!!! caution
```

Comment:
```suggestion
!!! warning
```

I'm using `warning` here, as `caution` is no longer supported by Material for MkDocs.